indico/modules/attachments/forms.py

```diff
@@ -101,10 +101,18 @@ class AttachmentFolderForm(IndicoForm):
     acl = AccessControlListField(_("Access control list"), [UsedIf(lambda form, field: form.protected.data)],
                                  groups=True, allow_external=True, default_text=_('Restrict access to this folder'),
                                  description=_("The list of users and groups allowed to access the folder"))
-    is_always_visible = BooleanField(_("Always Visible"), widget=SwitchWidget(),
+    is_always_visible = BooleanField(_("Always Visible"),
+                                     [HiddenUnless('is_always_hidden_in_fe', value=False), ],
+                                     widget=SwitchWidget(),
                                      description=_("By default, folders are always visible, even if a user cannot "
                                                    "access them. You can disable this behavior here, hiding the folder "
                                                    "for anyone who does not have permission to access it."))
+    is_always_hidden_in_fe = BooleanField(_("Always Hidden in Frontend"),
+                                          [HiddenUnless('is_always_visible', value=False), ],
```

Comment:
no need for the `, ` at the end of the list - only single-element tuples need this.

Reply:
Done